engine/src/main/java/org/terasology/rendering/openvrprovider/OpenVRStereoRenderer.java

```diff
@@ -0,0 +1,73 @@
+package org.terasology.rendering.openvrprovider;
+
+import jopenvr.JOpenVRLibrary;
+import org.lwjgl.opengl.EXTFramebufferObject;
+import org.lwjgl.opengl.GL11;
+
+import static org.lwjgl.opengl.EXTFramebufferObject.*;
+import static org.lwjgl.opengl.GL11.GL_DEPTH_COMPONENT;
+import static org.lwjgl.opengl.GL11.GL_TEXTURE_2D;
+
+/* This class is designed to manage the framebuffers for the headset. */
+public class OpenVRStereoRenderer {
+    public OpenVRProvider vrProvider;
+
+    // TextureIDs of framebuffers for each eye
+    private int eyeTextureIDs[] = new int[2];
+    private int eyeFBOIds[] = new int[2];
+
+    public OpenVRStereoRenderer(OpenVRProvider vrProviderIn, int lwidth, int lheight) {
+        vrProvider = vrProviderIn;
+        createRenderTexture(lwidth, lheight);
+    }
+
+    public void deleteRenderTextures() {
+        if (eyeTextureIDs[0] > 0) {
+            GL11.glDeleteTextures(eyeTextureIDs[0]);
+        }
+    }
+
+    public void createRenderTexture(int lwidth, int lheight) {
```

Comment:
As I mentioned earlier, I think I can just create the FBOs through Terasology's system and send their ID to OpenVR's compositor. I'll give it a shot anyway.